contrib/cmd/user-broker/user-broker.go

```diff
@@ -37,10 +42,39 @@ func init() {
 }
 
 func main() {
+	if err := run(); err != nil && err != context.Canceled && err != context.DeadlineExceeded {
+		glog.Fatalln(err)
```

Comment:
@ash2k Agree that it should exit 0, and that's doable with this code:

```go
glog.Info(err)
os.Exit(0)
```

That will achieve the logging that I requested and still exit 0. Thoughts?

Also, agree on `http.ErrServerClosed`